workflow_array_ephys/ingest.py

```diff
@@ -115,6 +134,38 @@ def ingest_sessions(session_csv_path='./user_data/sessions.csv', verbose=True):
         print('\n---- Successfully completed ingest_subjects ----')
 
 
+def ingest_events(recording_csv_path='./user_data/behavior_recordings.csv',
+                  block_csv_path='./user_data/blocks.csv',
+                  trial_csv_path='./user_data/trials.csv',
+                  event_csv_path='./user_data/events.csv',
+                  skip_duplicates=True, verbose=True):
+    csvs = [recording_csv_path, recording_csv_path,
+            block_csv_path, block_csv_path,
+            trial_csv_path, trial_csv_path, trial_csv_path,
+            trial_csv_path,
+            event_csv_path]
+    tables = [event.BehaviorRecording(), event.BehaviorRecording.File(),
+              trial.Block(), trial.Block.Attribute(),
+              trial.TrialType(), trial.Trial(), trial.Trial.Attribute(),
+              trial.BlockTrial(),
+              event.EventType(), event.Event()]
```

Comment:
I notice 9 files listed but 10 tables.  Was this intended?  Perhaps I am misunderstanding something.  Thanks.